client/styles/components/_hints.scss

```diff
@@ -0,0 +1,182 @@
+.CodeMirror-hints {
+  position: absolute;
+  z-index: 10;
+  overflow: hidden;
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+
+  -webkit-box-shadow: 0 0 18px 0 rgba(0, 0, 0, 0.16);
```

Comment:
You don't need to add the prefixes, this is automatically added with webpack.

Reply:
Resolved with other requested CSS changes!